_includes/scenarios/integrate/updating-identityattributes-of-yourself.md

```diff
@@ -1,17 +1,45 @@
+If you shared your old number with peers via enmeshed, you'll look for an uncomplicated way to let them know about this change.
+For this, you are given the possibility to succeed Attributes of your own.
+This will not only update their values, but also provide you and your peers with a coherent history of all past versions.
+How the Attribute succession works in detail depends on the type of Attribute.
 
-Attributes as such cannot be updated, but can be succeeded by a new Attribute. Thus, the value of an Attribute cannot be changed but overwritten by a new Attribute. We ensure that a coherent history of Attributes are available at all times on different parties.
+## Succeeding own IdentityAttributes
 
-Consider an IdentityA which shared its DisplayName to IdentityB, IdentityC, and IdentityD. IdentityA now would like to change the DisplayName and has to execute the following steps:
+In contrast to [RelationshipAttributes]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) you can [create IdentityAttributes]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) privately without sharing them with anyone.
+In order to [share an IdentityAttribute]({% link _docs_use-cases/use-case-consumption-share-an-identityattribute.md %}), you need to send an according [Request]({% link _docs_integrate/data-model-overview.md %}#request) to the peer.
+If they accept, a new [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) will be created at the peer's side.
+It has the same content like your private IdentityAttribute and, in addition, a `shareInfo` property.
+It stores information about the respective `peer`, the `sourceReference` to the private IdentityAttribute, whose content you copied, and a reference to the Request used to share the Attribute.
+Moreover, another copy will be created at your side, that has a `shareInfo`, too.
+Thus, sharing an IdentityAttribute will create an own shared IdentityAttribute copy for every peer you shared the Attribute with at your side and a peer shared IdentityAttribute copy for each peer at their side.
 
-- Succeed the LocalAttribute of the current DisplayName with the new DisplayName
-- Notify each peer individually that the DisplayName changed (this is optional, for every peer it can be decided if the peer receives the new Attribute or not)
+Now, let's consider the case that the value of your IdentityAttribute changes and you want to update it.
+To [succeed the (unshared) IdentityAttribute]({% link _docs_use-cases/use-case-consumption-succeed-an-identityattribute.md %}), a new version with the updated value will be created.
+In its `succeeds` property the predecessor is stored.
+It in turn is updated, too, to link to the newer version in its `succeededBy` property.
+Consequently, succeeding an Attribute, a double-linked list of all versions is created.
```

Comment:
Confusing: When succeeding _one_ attribute, a doubly linked list of _all_ versions is created. Maybe something more precise, like "_In short, the different versions of an attribute created by attribute successions are stored in a doubly linked list._"

And, at least as far as I'm aware, it's _doubly linked_, not _double-linked_.

Reply:
I found both translations, but wikipedia uses doubly linked list, so this will be our ground truth.